README.md

```diff
@@ -46,7 +46,10 @@ python -m openstef task <task_name>
 
 ## Reference Implementation
 A complete implementation including databases, user interface, example data, etc. is available at: https://github.com/OpenSTEF/openstef-reference
-![image](https://user-images.githubusercontent.com/18208480/127109029-77e09c97-8d06-4158-8789-4c1d5ecede61.png)
+
+Dashboard documentation can be found [here](https://raw.githack.com/OpenSTEF/openstef/feature-KTPS-1832-dashboard_doc/html/openstef_dashboard_doc.html).
```

Comment:
Mooie manier om uitleg over het dashboard te geven! 
Misschien dat het logischer is om de imagemap toe te voegen aan de readme van de organisatie (https://github.com/OpenSTEF) en linkjes daarnaar te plaatsen in openstef en openstef-reference

Reply:
@FrankKr Ja, dat is een goed idee. Probeerde het net toe te voegen maar momenteel heb ik geen write access tot `.github `repository. Is het mogelijk om mij access te geven?